RELEASE.md

```diff
@@ -2,8 +2,9 @@
 
 1. Create a PR "Preparing release X.Y.Z" against master or maintenance branch ([example](https://github.com/jaegertracing/jaeger/pull/543/files)) by updating CHANGELOG.md to include:
     * A new section with the header `<X.Y.Z> (YYYY-MM-DD)`
-    * A curated list of notable changes and links to PRs. Do not simply dump git log, select the changes that affect the users.
     * The section can be split into sub-section if necessary, e.g. UI Changes, Backend Changes, Bug Fixes, etc.
+    * Run `make changelog` and manually select notable changes that affect users. The incorrect pull request titles can be changed on Github and changelog regenerated.
+    * A curated list of notable changes and links to PRs. Do not simply dump git log, select the changes that affect the users. To obtain the list of all changes run `make changelog`
```

Comment:
I think this bullet point could be moved back to its original position with the new part at the end. The previous new bullet point I think is redundant.